fastmri/data/subsample.py

```diff
@@ -57,15 +63,27 @@ def __init__(self, center_fractions: Sequence[float], accelerations: Sequence[in
             accelerations: Amount of under-sampling. This should have the same
                 length as center_fractions. If multiple values are provided,
                 then one of these is chosen uniformly each time.
+            sync_center_fractions_accelerations: Whether to sync indicies
```

Comment:
typo: `indices`

Reply:
Simplify the description? Whether to consider cross combinations of `center_faction` and `acceleration` or something like this?